### DISCUSSION TITLE
Add icons for different eQ-3 devices

### DISCUSSION THREAD

**Discussion by Denis1983** [SukramJ/aiohomematic#1887] [Ideas]
Now only one icon for all eQ-3 devices:
![image](https://github.com/user-attachments/assets/cf934de9-0170-4fef-a8bb-41122ff53bd5)
So it is hard to visually distinguish them in the device list.

Please add icons for different eQ-3 devices
Example:
![image](https://github.com/user-attachments/assets/e17f0e91-5a56-4328-bcc5-87ad48271b73)
![image](https://github.com/user-attachments/assets/eb3639cc-679f-46ae-8ed5-d889b4207b36)
![image](https://github.com/user-attachments/assets/ee1e5618-fad7-46d3-9ef5-df6c1854e72c)
...

I could prepare the icons. (in *.svg fromat?)

**Comment by SukramJ** [comment]
Nette Idee, aber wo wird das in HA eingebaut?  Siehe [dev docs](https://developers.home-assistant.io/).
Abgesehen davon wäre mir die Pflege bei über 100 Gerätetypen zu aufwendig, aber du kannst gerne einen PR erstellen.

**Reply by Denis1983** [reply]
> aber wo wird das in HA eingebaut? Siehe [dev docs](https://developers.home-assistant.io/). 

Sorry I am not familiar HA dev yet, may be I will start some day.

> Abgesehen davon wäre mir die Pflege bei über 100 Gerätetypen zu aufwendig, aber du kannst gerne einen PR erstellen.

May be ony a certain amount of devices can be supported. Which are most often used.
HmIP-BROLL, HmIP-BDT, HmIP-BWTH, HmIP-PDT,...
And make the extention easier.

![image](https://github.com/user-attachments/assets/d7fe8eaa-d92d-4ea6-be94-4ed5fedb035e)

**Reply by SukramJ** [reply]
> Sorry I am not familiar HA dev yet, may be I will start some day.

This may be a good starting point. As far as i know there is no option to select a device icon.

> May be ony a certain amount of devices can be supported. Which are most often used.
HmIP-BROLL, HmIP-BDT, HmIP-BWTH, HmIP-PDT,...

The problem is that everyone here only thinks about their own things, and the rest is left to me.

**Reply by SukramJ** [reply]
Warum antwortest Du eigentlich auf Englisch, obwohl du aus Deutschland bist, und deine Geräte deutsche Namen haben?

**Reply by Denis1983** [reply]
>As far as i know there is no option to select a device icon.

:(, so that means feature request to home assistant core, right?

>The problem is that everyone here only thinks about their own things, and the rest is left to me.

Sure I can help to provide as much as possible icons, I just need the entry point, where I see that the feature is possible and works.

>Warum antwortest Du eigentlich auf Englisch, obwohl du aus Deutschland bist, und deine Geräte deutsche Namen haben?

I assume that not only germain speaking people would be interested here
Or homematic = only germain? :)

**Reply by SukramJ** [reply]
Da dich meine direkte Ansprache nicht interessiert finde ich es schon nervig!
Wenn Du in den Dev Docs nichts findest musst du wohl einen HA Feature Request erstellen.

**Reply by SukramJ** [reply]
Es macht defintiv Sinn wenn möglich auf deutsch zu wechseln:
Warum auf die handvoll Nicht-Deutsch-Sprechenden Rücksicht nehmen, wenn man dafür die hunderten Nicht-So-Gut-Englisch-Sprechenden ausschliesst.
Die Doku ist die Ausnahme, aber die kann man in einem rutsch mit Google Translate oder anderen Tools auf deutsch übersetzten lassen.

Just my 2ct.

**Reply by Denis1983** [reply]
> Da dich meine direkte Ansprache nicht interessiert finde ich es schon nervig! Wenn Du in den Dev Docs nichts findest musst du wohl einen HA Feature Request erstellen.

Ich habe durchgesucht, und ich glaube das Icon die Integration zugewieesen ist, ist auch das Icon für alle Geräte.
https://developers.home-assistant.io/docs/add-ons/presentation?_highlight=png#add-on-icon--logo

**Reply by Denis1983** [reply]
> Es macht defintiv Sinn wenn möglich auf deutsch zu wechseln: Warum auf die handvoll Nicht-Deutsch-Sprechenden Rücksicht nehmen, wenn man dafür die hunderten Nicht-So-Gut-Englisch-Sprechenden ausschliesst. Die Doku ist die Ausnahme, aber die kann man in einem rutsch mit Google Translate oder anderen Tools auf deutsch übersetzten lassen.
> 
> Just my 2ct.

Ich habe beide Sprachen in den Diskussionen gesehen, deswegen habe ich eine nach meiner Auswahl ausgewählt, ich glaube fast alle auf GitHub können einigermaßen Englisch.

Meine 2 Eurocents.

**Reply by SukramJ** [reply]
Das hier ist keine Add-On, sondern eine Integration. Das Icon, das für alle Geräte hinterlegt ist liegt im [Brands repository](https://github.com/home-assistant/brands/tree/master/custom_integrations/homematicip_local).

**Reply by SukramJ** [reply]
> Ich habe beide Sprachen in den Diskussionen gesehen, deswegen habe ich eine nach meiner Auswahl ausgewählt, ich glaube fast alle auf GitHub können einigermaßen Englisch.

Da sind meine Erfahrungen mit Homematic Usern anders, insbesondere wenn es um Problembeschreibungen geht.
